projects/ngx-sub-form/package-lock.json

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
Please remove this file entirely.

Only the package.json is needed here (as far as I'm aware). Unless that it'd also been used while reading peer deps? Ummmh unsure. Any particular reason you added this?

Reply:
Had just ran npm i after forming it.  Didn't think to remove it before push.